src/Entity/Redirect.php

```diff
@@ -30,6 +30,7 @@
  *       "delete" = "Drupal\redirect\Form\RedirectDeleteForm",
  *       "edit" = "Drupal\redirect\Form\RedirectForm"
  *     },
+ *     "views_data" = "Drupal\views\EntityViewsData",
```

Comment:
There is a custom list builder here for our list. Now that we have a view, we can drop that. Remove our custom class but keep the default EntityListBuilder (for operations support), similar to the default EntityViewsData.

Also, we need to depend on the views module in the info.yml file then.